roles/contiv_network/tasks/ovs.yml

```diff
@@ -1,54 +1,92 @@
 ---
 # This role contains tasks for installing ovs
-- name: get openstack kilo repo
+
+# Local install
+- name: copy ovs file (redhat)
+  copy: src={{ contiv_standalone_binaries }}/openvswitch-2.5.0-2.el7.x86_64.rpm dest=/tmp/openvswitch-2.5.0-2.el7.x86_64.rpm
+  when: ansible_os_family == "RedHat" and contiv_network_local_install == True
+
+
+- name: install ovs (redhat)
+  yum: name=/tmp/openvswitch-2.5.0-2.el7.x86_64.rpm state=present
+  when: ansible_os_family == "RedHat" and contiv_network_local_install == True
+  tags:
+    - prebake-for-dev
+
+
+- name: copy ovs common file (debian)
+  copy: src={{ contiv_standalone_binaries }}/ovs-common.deb dest=/tmp/ovs-common.deb
+  when: ansible_os_family == "Debian" and contiv_network_local_install == True
+
+- name: copy ovs switch file (debian)
+  copy: src={{ contiv_standalone_binaries }}/ovs-switch.deb dest=/tmp/ovs-switch.deb
+  when: ansible_os_family == "Debian" and contiv_network_local_install == True
+
+
+
+- name: install ovs-common (debian)
+  apt: "deb=/tmp/ovs-common.deb"
+  when: ansible_os_family == "Debian" and contiv_network_local_install == True
+  tags:
+    - prebake-for-dev
+
+- name: install ovs (debian)
+  apt: "deb=/tmp/ovs-switch.deb"
+  when: ansible_os_family == "Debian" and contiv_network_local_install == True
+  tags:
+    - prebake-for-dev
+
+# Remote install
+
+- name: get openstack ocata repo
   get_url:
-    url: https://repos.fedorapeople.org/repos/openstack/openstack-kilo/rdo-release-kilo-2.noarch.rpm
-    dest: /tmp/rdo-release-kilo-2.noarch.rpm
+    url: https://repos.fedorapeople.org/repos/openstack/openstack-ocata/rdo-release-ocata-2.noarch.rpm
```

Comment:
If we are using public repos we need to track changes to them. Given this is their latest version, it will hopefully be not too soon.